home-manager/default.nix

```diff
@@ -12,11 +12,18 @@ let
 
 in
 
-pkgs.stdenv.mkDerivation {
-  name = "home-manager";
-
-  buildCommand = ''
-    install -v -D -m755 ${./home-manager} $out/bin/home-manager
+pkgs.runCommand
+  "home-manager"
+  {
+    meta = with pkgs.stdenv.lib; {
+      description = "A user environment configurator";
+      maintainers = [ maintainers.rycee ];
+      platforms = platforms.unix;
+      license = licenses.mit;
+    };
+  }
```

Comment:
does it make sense to build this only locally? As I mentioned in my comment below, it's not set by default.